package.json

```diff
@@ -22,6 +22,9 @@
     "categories": [
         "Azure"
     ],
+    "extensionKind": [
```

Comment:
The most important part of this PR is making "ui" the default, which I agree with.

The other piece is this will completely block people from setting it as workspace. Are we 100% sure no one will want that? Seems pretty easy to add "workspace" at the end of the this list instead of blocking that scenario

Reply:
I think the primary concern is a user creating VMs through the extension while remotely connected to a VM, which would then add the new SSH keys on the remote machine rather than their local machine.

That being said, I'm not really sure why a user would want to create a VM while they're remotely connected but then again, someone reported this: 
> just install vscode insider and Remote SSH plugin. Try sshing into linux VM using a Windows VM. - https://github.com/microsoft/vscode-azurevirtualmachines/issues/104

So I dunno. 🤷‍♀️

Reply:
Right I get the concern, which is why I agree "ui" should be our default.

I'm saying you change "extensionKind" to be ["ui", "workspace"], which means all users run as "ui" by default, but we still support "workspace" if a user explicitly changes their settings and makes us run as "workspace".